test/functional/ui/screen.lua

```diff
@@ -37,10 +37,10 @@
 -- Tests will often share a group of extra attribute sets to expect(). Those can be
 -- defined at the beginning of a test:
 --
---    screen:add_extra_attr_ids {
+--    screen:add_extra_attr_ids({
 --      [100] = { background = Screen.colors.Plum1, underline = true },
 --      [101] = { background = Screen.colors.Red1, bold = true, underline = true },
---    }
+--    })
```

Comment:
stylua doesn't enforce this does it?

Reply:
It doesn't but I changed it for consistency with what `_print_snapshot()` spits out for the `expect()` call. Do we have a preference? (Mine is with `()`).

Reply:
`expect{` is 3x more common than `expect({` so I guess our preference is `expect{`.

Reply:
On the other hand `expect([[` is 1000x more common than `expect [[`.

Reply:
My sense from observing other core contributors is that `expect{` is preferred

Reply:
Because that's the current output of `snapshot_util()`...